### ISSUE TITLE
How do you use this?

### ISSUE THREAD

**Issue by Froznic** [sibvic/watchitonce#1]
I'm able to run it fine but the way your code is, right now, it comes up with the error "No files to play".  How do you normally use this for it to play stuff?  I like the idea of it, but struggling to figure out how to make it work for me.

**Comment by sibvic** [comment]
Just type wio.exe in the directory with videos. It scans the current directory for video files and starts playing. Maybe you have files with "unknown" extensions (by default it scans for "_.mkv", "_.avi", "_.mp4", "_.webm", "_.wmv", "_.vob"), try to specify the --extensions key, like that:
wio --extensions "*.mp3".
Let me know whether you solve your problem.

**Comment by Froznic** [comment]
Getting an exception error now, pretty sure it's something to do with the version of the dll's I'm using.  I have the latest stable release of vlc and that's where I pulled the dll's from so possibly that's where the problem is coming from.  Here's the exact message I got from debugging it.

"Failed to initialize libVLC. Possible reasons : Some of the arguments may be incorrect. VLC dlls' version mismatch."

I grabbed nLog version 3.0  Should I use a different one?
